includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1376,6 +1376,36 @@ public function get_terminal_readers() {
 		return $this->request( [], self::TERMINAL_READERS_API, self::GET );
 	}
 
+	/**
+	 * Registers a card reader to a terminal.
+	 *
+	 * @param string  $location           The location to assign the reader to.
+	 * @param string  $registration_code  A code generated by the reader used for registering to an account.
+	 * @param ?string $label              Custom label given to the reader for easier identification.
+	 *                                    If no label is specified, the registration code will be used.
+	 * @param ?array  $metadata           Set of key-value pairs that you can attach to the reader.
+	 *                                    Useful for storing additional information about the object.
+	 *
+	 * @return array Stripe terminal reader object.
+	 * @see https://stripe.com/docs/api/terminal/readers/object
+	 *
+	 * @throws API_Exception If an error occurs.
+	 */
+	public function register_terminal_reader( string $location, string $registration_code, string $label = null, array $metadata = null ) {
+		$request                      = [];
+		$request['location']          = $location;
+		$request['registration_code'] = $registration_code;
+
+		if ( isset( $label ) ) {
```

Comment:
`if ($label !== null )` here would improve readability.

Reply:
Fixed in fdc6305.